lendings/templates/lendings/return_book.html

```diff
@@ -1,19 +1,22 @@
 {% extends 'base.html' %}
 {% block title %}Return book: {{ item.get_title }}{% endblock %}

```

Comment:
You can replace this with the Title part of card

Reply:
I'd stick to using the page title for now, so we can improve the layout of the entire site at once once we have an idea for this.
Breaking with the standardised templates will mean a lot of extra work once we start tackling the outside template.